docs/usage/configuration-options.md

```diff
@@ -2280,6 +2280,12 @@ For example to apply a special label for Major updates:
 }
 ```
 
+<!-- prettier-ignore -->
+!!! warning
+    Packages that follow SemVer are allowed to make breaking changes in _any_ `0.x` version, even `patch` and `minor`.
+    Check if you're using any `0.x` package, and see if you need custom `packageRules` for it.
+    When setting up automerge for dependencies, make sure to stop accidental automerges of `0.x` versions.
```

Comment:
Should I link to this config example here?

https://docs.renovatebot.com/key-concepts/automerge/#automerge-non-major-updates

Reply:
sounds good